test/operations/generate/universalSeedMetadataTest.ts

```diff
@@ -36,18 +34,6 @@ describe("UniversalSeed metadata test", () => {
         validateUniversalSeedMetadata(md);
     });
 
-    it("supports metadata inheritance: one level", () => {
```

Comment:
Not necessary for this PR, but it may be a good idea to retain an "inherited" generator in the tests so we can retain tests for this functionality.